llama/sap.py

```diff
@@ -402,3 +403,27 @@ def generate_sap_control(sap_data_file: str, invoice_total: float) -> str:
     sap_control_file += "\n"
 
     return sap_control_file
+
+
+def get_and_update_sap_sequence_parameter(
+    ssm: SSM,
+    date: datetime,
+    sequence_type: str,
+    parameter_key: str,
+) -> str:
+    """Get SAP sequence parameter and update it according to specified values."""
+    old_sap_sequence = ssm.get_parameter_value(parameter_key)
+    new_sap_sequence = update_sap_sequence(old_sap_sequence, date, sequence_type)
+    return new_sap_sequence
+
+
+def update_sap_sequence(
```

Comment:
As discussed offline, this function should take the args it's currently getting, but rather than return them it should go ahead and update the parameter in SSM and return the response. The parameter type for this should be "StringList" and as in the function above, the parameter key should be generated from config.